spswarehouse/powerschool/calpads.py

```diff
@@ -0,0 +1,220 @@
+import time
+from spswarehouse.powerschool.powerschool import (
+                                            navigate_to_specific_state_report, 
+                                            download_latest_report_from_report_queue_reportworks, 
+                                            download_latest_report_from_report_queue_system,
+                                            switch_to_school,
+                                            powerschool_report_helper_type_in_element_by_id,
+                                            powerschool_report_helper_type_in_element_by_name,
+                                            powerschool_report_helper_select_visible_text_in_element_by_id,
+                                            powerschool_report_helper_select_visible_text_in_element_by_name,
+                                            powerschool_report_helper_click_element_by_id,
+                                            )
+
+from selenium.webdriver.chrome.webdriver import WebDriver
+
+def download_calpads_report_for_school(driver: WebDriver, school_full_name: str, report_name: str, school_subdistrict_name: str, 
```

Comment:
From a style perspective, a lot of these lines are too wide I think. My general frame is that if I can't see the whole line when I split screen, then we should do our best to make the line shorter.  This case has a very natural breakup, which is to put each argument on one line each.

Reply:
Made indentation changes. Will test and then resolve this.